src/scroll_accelerator/cli.py

```diff
@@ -0,0 +1,501 @@
+)
+from scroll_accelerator.daemon import (
+    daemon_log,
+    daemon_status,
+    install_daemon,
+    restart_daemon,
+    stop_daemon,
+    uninstall_daemon,
+)
+from scroll_accelerator.scroll_accelerator import (
+    ScrollAccelerator,
+    check_config,
+)
+
+# Logging
+
+logger = logging.getLogger(__name__)
+
+
+def init_logging(verbose: int = 0):
+    """Initialize logging.
+
+    Args:
+        verbose: Verbosity level.
+    """
+    if verbose <= 0:
+        format = "%(levelname)s: %(message)s"
+    else:
+        format = "%(asctime)s %(levelname)s: %(message)s"
+
+    logging.basicConfig(
+        format=format,
+        level=max(1, logging.WARNING - verbose * 10),
+    )
+
+
+# Config
+
+
+def get_default_config() -> dict:
+    """Default config for the application.
+
+    This is probably very subjective. But just to have some reasonable defaults,
+    which might be used.
+
+    Returns:
+        Default config for the application.
+    """
+    if sys.platform == "darwin":
+        # Darwin (MacOSX) already has builtin mouse scroll wheel acceleration,
+        # so we should apply much less extra acceleration.
+        config = DEFAULT_MACOS_CONFIG.copy()
+    else:
+        # All other common desktop platforms do not (Linux, Windows).
+        # (Android does, but we don't expect that here...)
+        config = DEFAULT_CONFIG.copy()
+
+    try:
+        check_config(**config)
+    except Exception as e:
+        assert False, f"Default config is invalid: {e}."
+
+    return config
+
+
+def get_command_line_config(args: argparse.Namespace) -> dict:
+    """Get config from command line arguments.
+
+    Args:
+        args: Command line arguments.
+
+    Returns:
+        Config from command line arguments.
+    """
+    default_config = get_default_config()
+    config = {}
+    for key in default_config.keys():
+        value = getattr(args, key)
+        if value is not None:
+            config[key] = value
+    return config
+
+
+def load_config() -> dict:
+    """Load config from yaml file.
+
+    Returns:
+        Config from file.
+
+    Raises:
+        RuntimeError: If the config file does not exist, is empty, or is invalid.
+    """
+    if not os.path.exists(CONFIG_FILE):
+        logger.warning(f"Config file {CONFIG_FILE} does not exist.")
+        return {}
+
+    with open(CONFIG_FILE, "r") as f:
+        config = yaml.safe_load(f)
+
+    # If the config file is empty, return an empty dict
+    if config is None:
+        logger.warning(f"Config file {CONFIG_FILE} is empty.")
+        return {}
+
+    # If the config file is not a dict, raise an error
+    if not isinstance(config, dict):
+        raise RuntimeError(f"Config file {CONFIG_FILE} is invalid.")
+
+    # Check that the config file contains only valid keys
+    default_config = get_default_config()
+    for key in config.keys():
+        if key not in default_config:
+            raise RuntimeError(f"Unknown config key in {CONFIG_FILE}: {key}")
```

Comment:
I think `RuntimeError` is usually used for different purpose in Python.
```suggestion
            raise ValueError(f"Unknown config key in {CONFIG_FILE}: {key}")
```